pkg/collector/corechecks/containerimage/processor.go

```diff
@@ -52,13 +53,27 @@ func (p *processor) processRefresh(allImages []*workloadmeta.ContainerImageMetad
 }
 
 func (p *processor) processImage(img *workloadmeta.ContainerImageMetadata) {
+	var lastCreated *timestamppb.Timestamp = nil
 	layers := make([]*model.ContainerImage_ContainerImageLayer, 0, len(img.Layers))
 	for _, layer := range img.Layers {
+		var created *timestamppb.Timestamp = nil
+		if layer.History.Created != nil {
+			created = timestamppb.New(*layer.History.Created)
+			lastCreated = created
```

Comment:
Quick question, is it expected that the Layers are always ordered in ascending order of `created` time?

Reply:
That makes sense. Thanks for clarifying it, Lénaïc! I jus wanted to make sure that we didn't need a check to override `lastCreated` only if `created` is more recent.

Reply:
I double-checked on a live system with the debugger and I confirm that layers are sorted is ascending order of `created` time:

```
(dlv) p layers[0].History.Created.Seconds
1674108677
(dlv) p layers[1].History.Created.Seconds
1674125890
(dlv) p layers[2].History.Created.Seconds
1674125896
(dlv) p layers[3].History.Created.Seconds
1674202789
(dlv) p layers[4].History.Created.Seconds
1674202790
(dlv) p lastCreated
*google.golang.org/protobuf/types/known/timestamppb.Timestamp {
        state: google.golang.org/protobuf/internal/impl.MessageState {
                NoUnkeyedLiterals: google.golang.org/protobuf/internal/pragma.NoUnkeyedLiterals {},
                DoNotCompare: google.golang.org/protobuf/internal/pragma.DoNotCompare [],
                DoNotCopy: google.golang.org/protobuf/internal/pragma.DoNotCopy [],
                atomicMessageInfo: *google.golang.org/protobuf/internal/impl.MessageInfo nil,},
        sizeCache: 0,
        unknownFields: []uint8 len: 0, cap: 0, nil,
        Seconds: 1674202790,
        Nanos: 888942295,}
```